src/python/WMCore/RequestManager/RequestDB/Settings/RequestStatus.py

```diff
@@ -35,19 +30,14 @@
 
 NextStatus = {
  "new" : ["new", "testing-approved", "assignment-approved", "rejected"],
- "testing-approved" : ["testing-approved", "testing", "test-failed", "aborted"],
- "testing" : ["testing", "tested", "test-failed", "aborted"],
- "tested" : ["tested", "assignment-approved","failed", "rejected", "aborted"],
- "test-failed" : ["test-failed", "testing-approved", "rejected", "aborted"],
  "assignment-approved" : ["assignment-approved", "assigned", "rejected"],
  "assigned" : ["assigned","acquired", "aborted", "failed"],
- "acquired" : ["running-open", "failed", "completed", "acquired", "aborted"],
- "running" : ["completed", "aborted", "failed", "epic-FAILED"],
- "running-open" : ["running-closed", "aborted", "failed", "epic-FAILED"],
- "running-closed" : ["force-complete", "completed", "aborted", "failed", "epic-FAILED"],
+ "acquired" : ["running-open", "completed", "acquired", "aborted"],
+ "running" : ["completed", "aborted"],
+ "running-open" : ["running-closed", "aborted"],
+ "running-closed" : ["force-complete", "completed", "aborted"],
  "force-complete" : ["completed"],
- "failed" : ["failed", "testing-approved", "assigned", "rejected"],
- "epic-FAILED" : ["epic-FAILED"],
+ "failed" : ["failed", "assigned", "rejected"],
```

Comment:
I remember there was some problem doing that in ReqMgr2. I am not sure what it was. But for the safe side I left it there.